wiki/en/en-Installation-for-Windows.md

```diff
@@ -8,10 +8,10 @@ permalink: "/wiki/Installation-for-Windows"
 # Installation for Windows
 Make sure you read the [Getting Started](Getting-Started) page.
 1. **Download and install an ASIO Driver**. It is recommended to use a sound card/interface with a native ASIO driver. If you don't have one (especially internal sound cards), install [this free ASIO driver (ASIO4ALL)](https://www.asio4all.org){: target="_blank" rel="noopener noreferrer"} before installing Jamulus.
-1. [Download Jamulus](https://sourceforge.net/projects/llcon/files/latest/download){: .button target="_blank" rel="noopener noreferrer"}.
-1. **Install Jamulus** If you get a warning notice from SmartScreen, click on "More info" and "Run anyway" to install Jamulus. This is needed since we don't pay for code signing yet.
+1. [Download Jamulus](https://sourceforge.net/projects/llcon/files/latest/download){: .button target="_blank" rel="noopener noreferrer"}
```

Comment:
Add button like on macOS.

Discussion:

Is this too prominent? Do we need to focus the ASIO driver too (but not everyone needs ASIO4ALL)

Reply:
Agree on the button. I don't think it's too prominent really, but the ASIO issue is tricky. I think we have it about right (although see my comment on possibly clarifying what "internal sound card" means).

Reply:
https://github.com/corrados/jamulus/issues/792#issuecomment-752946745

Maybe we could add a statement to the installer.